grafana-plugin/src/containers/IntegrationForm/IntegrationForm.tsx

```diff
@@ -288,47 +288,47 @@ export const IntegrationForm = observer(
             </div>
 
             <Controller
-              name={FormFieldKeys.ServiceNowUrl}
+              name={'additional_settings.instance_url'}
               control={control}
               rules={{ required: 'Instance URL is required', validate: validateURL }}
               render={({ field }) => (
                 <Field
                   key={'InstanceURL'}
                   label={'Instance URL'}
-                  invalid={!!errors[FormFieldKeys.ServiceNowUrl]}
-                  error={errors[FormFieldKeys.ServiceNowUrl]?.message}
+                  invalid={!!errors['additional_settings']?.['instance_url']}
```

Comment:
Can we use in every place `errors.additional_settings.instance_url` syntax? The reason is that because we have some rules disabled in TS, things like `errors['additional_settings']?.['passwordSomethingWrongHere']` doesn't throw compilation error so if the API contract changes, we will not be notified about such places that needs a change by TS